scheduler/conf.c

```diff
@@ -939,6 +939,17 @@ cupsdReadConfiguration(void)
 
     cupsdSetString(&ServerName, temp);
 
+    /*
```

Comment:
Some coding style issues here - comment should be indented one less column (so asterisks line up with code) and open braces are on the following line, e.g.:

```c
   /*
    * ServerName was updated.  Since a '%s' in the ErrorLog value
    * substitutes the server name we need to close ErrorFile.
    */

    if (ErrorFile != NULL && ErrorFile != LogStderr)
    {
      cupsFileClose(ErrorFile);
      ErrorFile = NULL;
    }
```

Reply:
Also, this isn't the only place where ServerName is set...